.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.11.4
+      - image: circleci/golang:1.13
```

Comment:
`golang:1.11.4` is still used in `test` job below

Reply:
done